### PR TITLE
Docs29 mac 2

### PR DESCRIPTION
**Pull request by smoe**
Couple of changes while translating against branch 2.9

### CODE REVIEW

File: docs/man/man1/xhc-hb04.1 [hunk 1]
```diff
@@ -108,91 +100,83 @@ Example:
...
-inactive (sleeping) message.
+(bit out) \fIxhc\-hb04.sleeping\fR True when the driver receives a pendant inactive (sleeping) message.
 .TP
 (bit out) \fIxhc\-hb04.jog.enable\-off\fR True when the pendant rotary
 selector switch is in the OFF position or when the pendant is sleeping.
 .TP
-(bit out) \fIxhc\-hb04.enable\-[xyza]\fR True when the pendant rotary
-selector switch is in the [xyza] position and not sleeping.
+(bit out) \fIxhc\-hb04.enable\-[xyza]\fR
+True when the pendant rotary selector switch is in the [xyza] position and not sleeping.
```
Comment (hansu) [inline docs/man/man1/xhc-hb04.1:151]: Why are these line break changes necessary?

Comment (smoe) [reply docs/man/man1/xhc-hb04.1:151]: I got a sentence starting with "selector switch is in the [xyza] position and not sleeping." to translate in weblate. All those line breaks are about keeping those parts of a paragraph on a single that that make no sense if read without context. Also the highlighting is often broken into multiple llines, which is not compatible with the auto-translators.

File: docs/man/man1/xhc-hb04.1 [hunk 2]
```diff
@@ -108,91 +100,83 @@ Example:
...
+A 1 pulse on this pin changes the stepsize to the next lower stepsize in the stepsize sequence specified in the xhc\-hb04 (loadusr) command.
 
 .SH Input Pins (to the pendant LCD display)
 .TP
-(float in) \fIxhc\-hb04.[xyza].pos\-absolute\fR Absolute position display.
-(typically connect to: halui.axis.N.pos\-feedback). The LCD display
-for pos\-absolute is fixed format with a sign, 4 number digits and 3
-fraction digits (+XXXX.XXX), require: \-9999.999 <= value <= 9999.999.
+(float in) \fIxhc\-hb04.[xyza].pos\-absolute\fR
+Absolute position display.  (typically connect to: halui.axis.N.pos\-feedback).
```
Comment (hansu) [inline docs/man/man1/xhc-hb04.1:112]:
```suggestion
(float in) \fIxhc\-hb04.[xyza].pos\-absolute\fR Absolute position display. 
(typically connect to: halui.axis.N.pos\-feedback).
```

**Review by hansu — COMMENTED** [review]
The line breaks are not really consistent. 
For some, the description starts the line after the pin name, for some in the same line like line 132, 135, 145.
Please go through these line breaks again. Thanks!

File: docs/man/man1/xhc-hb04.1 (hunk 1 above)
Comment (hansu) [reply docs/man/man1/xhc-hb04.1:151]: ok

File: docs/man/man1/xhc-hb04.1 (hunk 2 above)
Comment (smoe) [reply docs/man/man1/xhc-hb04.1:112]: I just checked, the line breaks indeed have an effect on how the text is rendered. IMHO the pin names are too difficult to read when the description starts right afterward, so I decided for the line break after the pin. The current version is in 2.9 is inconsistent on this already.

**Comment by smoe** [comment]
Moved all pin descriptions to second line. I suggest to revisit this situation at some point when we address all those man pages.

**Comment by smoe** [comment]
@hansu resolved conflicts

**Comment by hansu** [comment]
Thanks

**Pull request merged**